example/app-framework/app/timer.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+import * as timer from '../lib/timer';
+
+/* To use variables in the callback function:
+    1. set variables as global variables.
+    2. use closure.
+   Here, we take 1.
+ */
```

Comment:
I think we can use closure since we support it

Reply:
Yes, use closure is better.